packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -80,17 +85,27 @@ class EnumerationWidget extends React.Component {
 			onClick: this.onDeleteItem.bind(this),
 		}];
 		this.defaultHeaderActions = [{
-			label: 'Add item',
-			icon: 'talend-plus',
-			id: 'add',
-			onClick: this.changeDisplayToAddMode.bind(this),
-		}, {
 			disabled: false,
 			label: 'Search for specific values',
 			icon: 'talend-search',
 			id: 'search',
 			onClick: this.changeDisplayToSearchMode.bind(this),
+		}, {
+			label: 'Add item',
+			icon: 'talend-plus',
+			id: 'add',
+			onClick: this.changeDisplayToAddMode.bind(this),
 		}];
+
+		if (this.allowImport) {
```

Comment:
I think this is a bit confusing (if a developper add a new element in `defaultHeaderActions`, the issue could be pretty difficul to find). What do you think about the following ? : 

```
this.defaultHeaderActions = [{
	label: 'Add item',
	icon: 'talend-plus',
	id: 'add',
	onClick: this.changeDisplayToAddMode.bind(this),
}];

if (this.allowImport) {
	this.defaultHeaderActions.push({
		label: 'Import values from a file',
		icon: 'talend-download',
		id: 'upload',
		onClick: this.simulateClickInputFile.bind(this),
	});
} else {
	this.defaultHeaderActions.push({
		label: 'Add item',
		icon: 'talend-plus',
		id: 'add',
		onClick: this.changeDisplayToAddMode.bind(this),
	});
}
```